app/views/users/_form.html.erb

```diff
@@ -1,6 +1,7 @@
 
 <%= form_for(@resource) do |f| %>
   <%= f.text_field :user_name %>
+  <%= f.check_box :mentor, value: @resource.mentor %>
```

Comment:
Is the value attribute necessary here? Wouldn't that be automatically filled in like the other fields?